plugins/catalog-backend/src/search/DefaultCatalogCollator.ts

```diff
@@ -81,6 +81,23 @@ export class DefaultCatalogCollator implements DocumentCollator {
     return formatted.toLowerCase();
   }
 
+  protected isUserEntity(entity: Entity): entity is UserEntity {
+    return entity.kind === 'User';
+  }
+
+  protected getDocumentText(entity: Entity): string {
+    let documentText = entity.metadata.description || '';
+    if (this.isUserEntity(entity)) {
+      if (entity.spec?.profile?.displayName && entity.metadata.description) {
+        // combine displayName and description
+        documentText = `${entity.spec?.profile?.displayName} : ${entity.metadata.description}`;
+      } else {
+        documentText = entity.spec?.profile?.displayName || documentText;
+      }
+    }
+    return documentText;
+  }
```

Comment:
as documentText is already set to description above, do you think this can shorten to something like

```ts
if (this.isUserEntity(entity)) {
    if (entity.spec?.profile?.displayName) {
         // add displayName, separate by colon
         documentText = documentText.concat(' : ', entity.spec?.profile?.displayName);
    } 
}
return documentText;
```

Reply:
That could work but it would end up looking like: `description : full name`, when I think it makes more sense to have the user's name first? `full name : description`. What do you think?

Reply:
I think that makes sense, and for that I guess you could just switch them around. 

```ts
if (this.isUserEntity(entity)) {
    if (entity.spec?.profile?.displayName) {
         // add displayName, separate by colon
         const displayName = entity.spec?.profile?.displayName
         documentText = displayName.concat(' : ', documentText);
    } 
}
return documentText;
```

Not a change that is required, but just a way to get rid of the entity.metadata.description check again! But totally optional. And the changes looks 💯 to me, leaving it to reviewers for approval!

Reply:
I can use .concat, but I am not sure I can get rid of the check for description... otherwise might end up with something like `full name : ` (if there is a displayName but no description). I could check for documentText not being empty instead of entity.metadata.description to shorten it though :)